### PR TITLE
Explicitly closing journal on rotate

### PR DESCRIPTION
**Pull request by ewolinetz**
Addresses https://github.com/reevoo/fluent-plugin-systemd/issues/34#issuecomment-380221532

Cherry pick of https://github.com/reevoo/fluent-plugin-systemd/pull/43

### CODE REVIEW

**Comment by errm** [comment]
In principle I am fine with this ... but please fixup the Gemfile so our CI will build ...

File: fluent-plugin-systemd.gemspec [hunk 1]
```diff
@@ -23,6 +23,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "test-unit", "~> 2.5"
   spec.add_development_dependency "reevoocop"
 
-  spec.add_runtime_dependency "fluentd", "~> 0.12"
+  spec.add_runtime_dependency "fluentd", "~> 0.13"
```
Comment (errm) [inline fluent-plugin-systemd.gemspec]: In fact I don't even understand what this is for there is no fluentd 0.13 please drop this change...

Comment (errm) [reply fluent-plugin-systemd.gemspec]: #43 changed the following line to:

```
spec.add_runtime_dependency "systemd-journal", "~> 1.3"
```

I think that is the change you want here ...

Comment (ewolinetz) [reply fluent-plugin-systemd.gemspec]: Thanks, that's definitely the line I wanted to change....

**Comment by errm** [comment]
Thanks I have merged your change and released version: 0.0.10

**Pull request closed without merging**